CONTRIBUTING.md

```diff
@@ -37,6 +37,23 @@ it in the commit message (`Fixes #42.`).
 
 [mailmap]: https://github.com/git/git/blob/master/Documentation/mailmap.txt
 
+Setup
+-----
+
+1. Install dependencies
+
+Cram is used for tests:
+
+    pip install cram
+
+The mustache gem is used for building the html pages.
+
+    gem install mustache
+
+2. Start hacking
+
+Just remember to run `./configure` before any `make` commands.
```

Comment:
Right, there won't be a `Makefile` unless `configure` is run first. But it shouldn't be needed every time; for example, if you run `make distcheck`, you can just run `make check` after without running `configure`.

Maybe a link to http://www.gnu.org/software/automake/manual/html_node/Autotools-Introduction.html will be sufficient to get people started? Or something like this:

https://developer.gnome.org/anjuta-build-tutorial/stable/figures/autotools-process.png.en